server.js

```diff
@@ -63,6 +63,15 @@ app.use('/data/commodities.json', express.static(__dirname + '/data/commodities.
 // get the data decorator
 var data = require('./data/decorator');
 
+if (process.env.HTTP_BASIC_AUTH) {
+  var bits = process.env.HTTP_BASIC_AUTH.split(':', 2);
```

Comment:
For this particular line, it'd be good to have some link, or description, of how you know `HTTP_BASIC_AUTH` is set to the `username:password` value (and ideally, why naively splitting on `:` is known to be acceptable).